### ISSUE TITLE
Officially support Android Firefox

### ISSUE THREAD

**Issue by Squish8294** [tom-james-watson/old-reddit-redirect#75]
Hey there. Old Reddit Redirect is currently not compatible with Android Firefox ver 110.0.1

It's been this way for quite some time, it started when they rolled Firefox Quantum to Android.

Sorry I can't be more helpful when opening this issue. If there's more information I can provide please let me know.

**Comment by tom-james-watson** [comment]
Until installing arbitrary extensions is possible without having to jump through tonnes of hoops, this is not really a priority

**Comment by francium** [comment]
They're finally doing it https://blog.mozilla.org/addons/2023/08/10/prepare-your-firefox-desktop-extension-for-the-upcoming-android-release/

Should we reopen this issue?

**Comment by tom-james-watson** [comment]
Yeah, it looks like potentially there will need to be some specific support added to handle mobile. Need to test this out some more.

**Comment by tom-james-watson** [comment]
I just tested on Firefox Nightly with a custom collection and the extension already works perfectly, as far as I can tell. Once official support is there for arbitrary addons, people should just be able to add it without problem. Going to close this out!

**Comment by Squish8294** [comment]
Thank you! <3 much love for re-visiting this, and thanks so much for your efforts!
